features/gba-dev/fields-mapping-gba.csv

```diff
@@ -0,0 +1,179 @@
+adressering.inOnderzoek.datumIngangOnderzoekPersoon,persoonInOnderzoek
+adressering.inOnderzoek.datumIngangOnderzoekVerblijfplaats,verblijfplaats.inOnderzoek
+adressering.inOnderzoek.gebruikInLopendeTekst,persoonInOnderzoek|partners.inOnderzoek
+adressering.inOnderzoek.land,verblijfplaats.inOnderzoek
+adressering.land,verblijfplaats.land
+adressering.land.code,verblijfplaats.land.code
+adressering.land.omschrijving,verblijfplaats.land.omschrijving
+geboorte.inOnderzoek,persoonInOnderzoek
+geboorte.inOnderzoek.datum,persoonInOnderzoek
+geboorte.inOnderzoek.datumIngangOnderzoek,persoonInOnderzoek.datumIngangOnderzoek
+geboorte.inOnderzoek.land,persoonInOnderzoek
+geboorte.inOnderzoek.plaats,persoonInOnderzoek
+immigratie.indicatieVestigingVanuitBuitenland,immigratie.datumVestigingInNederland
+immigratie.inOnderzoek,verblijfplaats.inOnderzoek
+immigratie.inOnderzoek.datumIngangOnderzoek,verblijfplaats.inOnderzoek.datumIngangOnderzoek
+immigratie.inOnderzoek.datumVestigingInNederland,verblijfplaats.inOnderzoek
+immigratie.inOnderzoek.indicatieVestigingVanuitBuitenland,verblijfplaats.inOnderzoek
+immigratie.inOnderzoek.landVanwaarIngeschreven,verblijfplaats.inOnderzoek
+immigratie.inOnderzoek.vanuitVerblijfplaatsOnbekend,verblijfplaats.inOnderzoek
+immigratie.vanuitVerblijfplaatsOnbekend,immigratie.landVanwaarIngeschreven
+inOnderzoek,persoonInOnderzoek|gezagInOnderzoek|verblijfplaats.inOnderzoek
+inOnderzoek.burgerservicenummer,persoonInOnderzoek
+inOnderzoek.datumIngangOnderzoekGemeente,verblijfplaats.inOnderzoek.datumIngangOnderzoek
+inOnderzoek.datumIngangOnderzoekGezag,gezagInOnderzoek.datumIngangOnderzoek
+inOnderzoek.datumIngangOnderzoekPersoon,persoonInOnderzoek.datumIngangOnderzoek
+inOnderzoek.datumInschrijvingInGemeente,verblijfplaats.inOnderzoek
+inOnderzoek.gemeenteVanInschrijving,verblijfplaats.inOnderzoek
+inOnderzoek.geslacht,persoonInOnderzoek
+inOnderzoek.indicatieCurateleRegister,gezagInOnderzoek
+inOnderzoek.indicatieGezagMinderjarige,gezagInOnderzoek
+inOnderzoek.leeftijd,persoonInOnderzoek
+kinderen.geboorte.inOnderzoek,kinderen.inOnderzoek
+kinderen.geboorte.inOnderzoek.datum,kinderen.inOnderzoek
+kinderen.geboorte.inOnderzoek.datumIngangOnderzoek,kinderen.inOnderzoek.datumIngangOnderzoek
+kinderen.geboorte.inOnderzoek.land,kinderen.inOnderzoek
+kinderen.geboorte.inOnderzoek.plaats,kinderen.inOnderzoek
+kinderen.inOnderzoek.burgerservicenummer,kinderen.inOnderzoek
+kinderen.naam.inOnderzoek,kinderen.inOnderzoek
+kinderen.naam.inOnderzoek.adellijkeTitelPredicaat,kinderen.inOnderzoek
+kinderen.naam.inOnderzoek.datumIngangOnderzoek,kinderen.inOnderzoek.datumIngangOnderzoek
+kinderen.naam.inOnderzoek.geslachtsnaam,kinderen.inOnderzoek
+kinderen.naam.inOnderzoek.voorletters,kinderen.inOnderzoek
+kinderen.naam.inOnderzoek.voornamen,kinderen.inOnderzoek
+kinderen.naam.inOnderzoek.voorvoegsel,kinderen.inOnderzoek
+kinderen.naam.voorletters,kinderen.naam.voornamen
+leeftijd,kinderen.geboorte.datum
```

Comment:
Moet dit niet kinderen.leeftijd zijn?

Reply:
andersom: leeftijd,geboorte.datum

kinderen.leeftijd is verwijderd uit de API